proto/cosmos/accounts/v1/accounts.proto

```diff
@@ -0,0 +1,4 @@
+syntax="proto3";
+
+package cosmos.accounts.v1;
+option go_package = "cosmossdk.io/x/accounts/v1";
```

Comment:
Just realized, the types here should just be generated in the root of the module: https://docs.cosmos.network/v0.50/building-modules/structure#structure, without that `/v1`.
Or is this on purpose for handling multiple future versions?

Reply:
i'm not clear on this, I followed the structure of gov: https://github.com/cosmos/cosmos-sdk/tree/main/proto/cosmos/gov/v1 which has cosmos/gov/v1,

Reply:
i followed the same structure as gov here: https://github.com/cosmos/cosmos-sdk/pull/17443

Reply:
Yeah the proto structure is good. My point is about the go packages.

Reply:
Yeah @julienrbrt, now I understood your comment better, but I think we should always prepend package versions to protobuf generated files, because if we do a bump to v2,  then would end up either moving the "old" v1 files in a v1 package (breaking change), or create a v2 package with v2 files (not breaking, but inconsistent).

I do not have a strong preference, I can also amend what I did.  LMK what you think.

Reply:
Right, that makes sense. We should maybe have it under a `/types` folder like gov for consistency with other modules.